### PR TITLE
update a few broken links from master->main transition

### PR DESCRIPTION
**Pull request by eteq**
Just what it says on the tin. Pinging @pllim for a review, since she caught the places I missed in spacetelescope/jdaviz#419

### CODE REVIEW

File: .circleci/builder/factory.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def obtain_options() -> argparse.Namespace:
     # Build Notebooks
     parser.add_argument('-c', '--notebook-collection-paths', type=str, default='')
     parser.add_argument('-n', '--notebook-category', type=str, default=None)
-    parser.add_argument('-r', '--remote-names', type=str, default='master')
+    parser.add_argument('-r', '--remote-names', type=str, default='main,master')
```
Comment (pllim) [inline .circleci/builder/factory.py:34]: Do we need to keep the old branch name?
```suggestion
    parser.add_argument('-r', '--remote-names', type=str, default='main')
```

**Review by pllim — COMMENTED** [review]
I didn't do anything fancy over at `jdaviz`; a recursive grep will smoke out the rest if you missed any.

`grep -r master .`

File: .circleci/builder/factory.py (hunk 1 above)
Comment (eteq) [reply .circleci/builder/factory.py:34]: I'm honestly not sure. I'm not clear on exactly what this is used for... it seemed safer to do it this way? :shrug:

**Comment by pllim** [comment]
Hmm weird, it has conflicts now?

**Comment by eteq** [comment]
Yeah, I dunno what happened there, but I fixed the conflicts...

File: .circleci/builder/factory.py (hunk 1 above)
Comment (pllim) [reply .circleci/builder/factory.py:34]: Is @jbcurtin working on the machinery of CircleCI here; perhaps he can comment?

Comment (jbcurtin) [reply .circleci/builder/factory.py:34]: Seems fine to me. 👍

**Comment by jbcurtin** [comment]
> Yeah, I dunno what happened there, but I fixed the conflicts...

I'm having trouble following what the issue is. The Build Machinery seems to be working fine with the updates you've made.

Looks like there are some new notebooks with new requirements for the CircleCI Executor which is where some of the errors are being through. 

I won't be able to look at those builds until two or three weeks from now. Would you like me to clean up those builds then?

**Comment by jbcurtin** [comment]
I've installed `nbcollection`. We no longer need to update the code here

**Pull request closed without merging**